.github/dependabot.yml

```diff
@@ -11,3 +11,15 @@ updates:
       actions-deps:
         patterns:
           - "*"
+
+  - package-ecosystem: "pip"
```

Comment:
Is it `pip` or `pip-compile`?

Reply:
it's `pip` -- https://docs.github.com/en/code-security/dependabot/ecosystems-supported-by-dependabot/supported-ecosystems-and-repositories#supported-ecosystems-and-repositories

<img width="853" alt="image" src="https://github.com/user-attachments/assets/35e41b7c-e1b4-4358-b650-b09d0e27b17c" />